package.json

```diff
@@ -72,7 +73,9 @@
     "jest": "^28.1.3",
     "lint-staged": "^13.0.3",
     "prettier": "^2.7.1",
-    "sonarjs": "^0.2.0"
+    "sonarjs": "^0.2.0",
+    "faker": "^5.5.3",
```

Comment:
not sure about this library that is dead (not maintain anymore) => https://www.npmjs.com/package/faker